tests/pytests/runtests.sh

```diff
@@ -206,11 +208,11 @@ setup_clang_sanitizer() {
 	fi
 
 	if [[ $SAN == addr || $SAN == address ]]; then
-		REDIS_SERVER=${REDIS_SERVER:-redis-server-asan-$SAN_REDIS_VER}
+		REDIS_SERVER=${REDIS_SERVER:-redis-server-asan-$SAN_REDIS_SUFFIX}
 		if ! command -v $REDIS_SERVER > /dev/null; then
 			echo Building Redis for clang-asan ...
-			$READIES/bin/getredis --force -v $SAN_REDIS_VER --own-openssl --no-run \
-				--suffix asan --clang-asan --clang-san-blacklist $ignorelist
+			runn $READIES/bin/getredis --force -v $SAN_REDIS_VER --own-openssl --no-run \
```

Comment:
`runn` typo?

Reply:
no typo :-)
this is a nice bash function from readies that runs a command and only outputs error if it fails. It can also invoke sudo if requested and reacts nicely to NOP=1 (no operation mode).